background/background.js

```diff
@@ -31,12 +31,12 @@ const LivemarkUpdater = {
     Settings.addChangeListener(this.handleSettingsChange);
 
     // We use this Map to mark feed items as visited.
-    // Map [int32 -> String | Array [String]]
+    // Map [int32 -> Set[String]]
```

Comment:
:slightly_frowning_face: I did this intentionally like this to avoid the overhead of another object